content/php-client/exception.md

```diff
@@ -0,0 +1,107 @@
+
+Every request done with the client can throw an HTTP exception.
+
+## HTTP exception
+
+The parent for of these HTTP exceptions is `Akeneo\Pim\Exception\HttpException`.
+You can get the request and the response that are responsible of this exception. Also, the HTTP response code and response message are available.
+ 
+```php
+try {
+    $client->getProductApi()->get('top');
+} catch (HttpException $e) {
+    // do your stuff with the exception
+    $requestBody = $e->getRequest()->getBody();
+    $responseBody = $e->getResponse()->getBody();
+    $responseMessage = $e->getResponseMessage();
+    $responseCode = $e->getResponseCode();
+}
+```
+
+ 
+Two types of exception inherits from this exception: server exception and client exception.
+
+## Server exception
+
+A server exception could be thrown by the client and means that the server failed to fulfill an apparently valid request.
+It's an HTTP code in the family 5xx.
+
+This exception is `Akeneo\Pim\Exception\ServerErrorHttpException`.
+
+```php
+try {
+    $client->getProductApi()->get('top');
+} catch (ServerHttpException $e) {
+    // do your stuff with the exception
+}
+```
+
+## Client exception
+
+A server exception could be thrown by the client and means is intended for situations in which the client seems to have errored.
+It's an HTTP code in the family 4xx.
+
+This exception is `Akeneo\Pim\Exception\ClientErrorHttpException`. It is subdivided in several specific exceptions.
+
+### Bad request exception
+
+This exception could be thrown if the request is not a valid JSON. It corresponds to the HTTP code 400.
+
+```php
+try {
+    $client->getProductApi()->get('top');
+} catch (BadRequestHttpException $e) {
+    // do your stuff with the exception
+}
+```
+
+::: info
+It should not occurred with the PHP client, because the JSON is generated from PHP array.
+:::
+
+### Not found exception
+
+This exception could be thrown if the requested resource does not exist. It corresponds to the HTTP code 404.
+
+```php
+try {
+    $client->getProductApi()->get('top');
+} catch (NotFoundHttpException $e) {
+    // do your stuff with the server exception
+}
+```
+
+### Unauthorized exception
+
+This exception could be thrown if you don't have the permission to access to the resource. It corresponds to the HTTP code 401.
+
+```php
+try {
+    $client->getProductApi()->get('top');
+} catch (UnauthorizedHttpException $e) {
+    // do your stuff with the server exception
+}
+```
+
+::: info
+It should not occurred as the PHP client automatically authenticates the request for you.
+:::
+
+### Unprocessable entity exception
+
+This exception could be thrown if the data are not valid. In this exception, an array of errors could be returned.
+It returns an empty array if there is no errors in the array.
+
+```php
+try {
+    $client->getProductApi()->create('top');
+} catch (UnauthorizedHttpException $e) {
+    // do your stuff with the server exception
+    foreach ($errors as $error) {
```

Comment:
I think you can repeat on each exception how to access to body/message/code/errors